internal/report/report.go

```diff
@@ -755,6 +755,15 @@ func printTraces(w io.Writer, rpt *Report) error {
 		}
 		sort.Strings(labels)
 		fmt.Fprint(w, strings.Join(labels, ""))
+
+		// Print any numeric labels for the sample
+		var numLabels []string
+		for k, v := range sample.NumLabel {
+			numLabels = append(numLabels, fmt.Sprintf("%10s:  %s\n", k, strings.Trim(fmt.Sprintf("%v", v), "[]")))
```

Comment:
Is %d happy to print an array? Hmm, if not maybe leave what you had with %v. It's weird but not enough to unroll into a loop here. The only string thing is that if the list is nil it would print nil which is weird. But apparently this should never happen.